src/app/project/beta/ontology-classes/ontology-class-item/ontology-class-item.component.ts

```diff
@@ -66,6 +66,18 @@ export class OntologyClassItemComponent implements OnInit {
                 this._getSearchCount();
             }
         ));
+
+        this.componentCommsSubscriptions.push(this._componentCommsService.on(
+            Events.resourceCreated, () => {
+                this._getSearchCount();
+            }
+        ));
+    }
+
+    ngOnDestroy(): void {
+        if (this.componentCommsSubscriptions !== undefined) {
```

Comment:
Is it necessary to check for undefined if you assign an empty array at the declaration?

Reply:
No, it's not necessary. I've removed it in [b0d2b01](https://github.com/dasch-swiss/dsp-app/pull/912/commits/b0d2b01577036aca6506a693129df69eb2fe94c8)